Sources/DeepLearning/Layer.swift

```diff
@@ -257,7 +332,10 @@ public extension Dense {
     }
 }
 
-/// A convolutional neural network layer.
+/// A 2D convolution layer (e.g. spatial convolution over images).
+///
+/// This layer creates a convolution kernel that is convolved with the layer input to produce a
+/// tensor of outputs. If activation is not `nil`, it is applied to the outputs as well.
```

Comment:
How about:
```suggestion
/// tensor of outputs. If activation is not `nil`, it is applied to the outputs.
```

Reply:
This is copied from Keras. The whole sentence should be deleted because we do not have `Optional` activation functions yet.